gridstatus/tests/test_ercot.py

```diff
@@ -10,6 +10,19 @@
 class TestErcot(BaseTestISO):
     iso = Ercot()
 
+    def test_get_sced_system_lambda(self):
```

Comment:
fix this test to make sure it passes. add test for  date="today" in addition to date='latest'